client/src/pages/Contact/Contact 2.js

```diff
@@ -0,0 +1,203 @@
+import "./contact.css";
```

Comment:
Is there a reason we are adding a 2nd contact page? I can see there is also `contact 2.css` and `ShareStar 2.js` files. Can they be removed from PR?